templates/dashboard/purchase_history.php

```diff
@@ -141,15 +141,15 @@
 									$items = ( new OrderModel() )->get_order_items_by_id( $order->id );
 									foreach ( $items as $item ) {
 										$course_id    = $item->id;
-										$object_title = '';
+										$object_title = get_the_title( $course_id );
```

Comment:
Since we are calling object title then why the variable name is **$course_id** it should be **$object_id**

Reply:
Because the immediate $course_id is for  single course, bundle order

Reply:
If there is nothing other than a course, we can use $course_title. Mixed variable names like $course_id & $object_title seems confusing.

Reply:
Already updated